docs/content/en/docs/reference/baremetal/bare-preparation.md

```diff
@@ -5,3 +5,71 @@ weight: 20
 description: >
   Set up a Bare Metal cluster to prepare it for EKS Anywhere
 ---
+After gathering hardware described in Bare Metal [Requirements]({{< relref "./bare-prereq" >}}), you need to prepare the hardware and create a CSV file describing that hardware.
+
+## Prepare hardware
+To prepare your computer hardware for EKS Anywhere, you need to connect your computer hardware and do some configuration.
+Once the hardware is in place, you need to:
+
+* Obtain IP addresses to assign to the IPMI and regular NIC interface on each machine in the cluster.
```

Comment:
The ipmi IP addresses should already be assigned. We will only need IP addresses to assign to the NICs. I would split this up into 2
```suggestion
* Obtain IP and MAC addresses for your machines' NICs.
* Obtain IP addresses for your machines' IPMI interfaces.
```

Reply:
Okay. Done.

Reply:
Done